### ISSUE TITLE
Continuing a run with Dark Rock after restarting the game causes console errors

### ISSUE THREAD

**Issue by alexlnkp** [OpenSauce04/EpiphanyIssues#91]
### Preamble

- [X] I either downloaded the mod by subscribing on the Steam Workshop, or I am using the official Gamebanana mirror
- [X] I have checked that I am running the latest version of Epiphany
- [X] I am playing on v1.7.9b of The Binding of Isaac: Repentance
- [X] I have tried installing either the 'Mod Error Container' mod or Repentogon before filing this issue

### What Happened?

i am playing tainted lazarus and reloaded the game on sheol's starting room.
got this error:
```
"MC_EVALUATE_CACHE" from "Epiphany" failed: ...463/scripts/epiphany/unlocks/unlocks_judas/dark_rock.lua:49: attempt to call a nil value (local 'func')
Stack Traceback:
  ...463/scripts/epiphany/unlocks/unlocks_judas/dark_rock.lua:49: in function 'GetDarkRockDmgMultiplier'
  ...463/scripts/epiphany/unlocks/unlocks_judas/dark_rock.lua:58: in function at line 52
```

the version of tainted lazarus i had when i clicked "continue" was the one with the dark rock

### Steps to Reproduce the Issue

- have dark rock as tainted lazarus (either the regular version or the awakened/flipped one)
- reload the game with the lazarus that has dark rock active/flipped to

### Diagnostics Information

[EpiphanyDiagnostics.tar.gz](https://github.com/OpenSauce04/EpiphanyIssues/files/14299391/EpiphanyDiagnostics.tar.gz)

### Additional Context

it doesn't seem to be a mod compatibility issue, however i do have repentogon installed, plus i also have birthright as tainted lazarus (could possibly lead to odd interactions?)

**Comment by OpenSauce04** [comment]
You are using the "Heart tokens for the lost" mod, which is known to cause major issues with other mods
Please remove that mod, restart the game, and see if the issue still occurs. If it does, please generate a new diagnostic file

**Comment by OpenSauce04** [comment]
I have managed to replicate this issue myself

**Comment by OpenSauce04** [comment]
This doesn't require Tainted Lazarus to trigger, it can happen with any character

**Comment by OpenSauce04** [comment]
This issue has been addressed internally, and a fix will be distributed in a future update. Thanks for the report

**Comment by OpenSauce04** [comment]
The fix has been released in 5.3.4
